var/spack/repos/builtin/packages/superlu/package.py

```diff
@@ -34,21 +35,86 @@ class Superlu(Package):
     url      = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_5.2.1.tar.gz"
 
     version('5.2.1', '3a1a9bff20cb06b7d97c46d337504447')
+    version('4.3', 'b72c6309f25e9660133007b82621ba7c')
 
-    depends_on('blas')
+    variant('blas', default=True, description='Build with external BLAS library')
```

Comment:
there is no consensus about this, but my opinion is that not packages should ever used internal BLAS. Imagine what happens if you have a graph of packages which are used to build something, all build as shared libraries, and each use it's own BLAS, some 32bit, some 64bit, some with threading, some without, etc.

So, i am not in favour of adding this variant.